src/Controls/tests/UITests/Tests/Issues/Issue20273.cs

```diff
@@ -0,0 +1,26 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.AppiumTests.Issues;
+
+public class Issue20273 : _IssuesUITest
+{
+	public override string Issue => "Previously selected item cannot be reselected";
+
+	public Issue20273(TestDevice device)
+		: base(device)
+	{ }
+
+    [Test]
+	public void PreviouslySelectedItemShouldBeReselected()
+	{
+		var firstItem = App.WaitForElement("First Item");
+		firstItem.Click();
+		App.WaitForElement("First Item");
+		firstItem.Click();
+		var numberOfNavigations = App.WaitForElement("numberOfNavigations").GetText();
+
+		Assert.AreEqual(int.Parse(numberOfNavigations!), 2);
```

Comment:
![image](https://github.com/dotnet/maui/assets/6755973/a7197448-823c-47b6-b697-00eebaba9ae3)
This test is failing.

Reply:
It should work now

Reply:
I have run this again on CI, and seems to keep failing. Will take a look locally.

Reply:
Sorry, get the same result 
![image](https://github.com/dotnet/maui/assets/6755973/d8a10294-d7b1-4f06-97ba-d2684ea8c731)
Let me know if can help in someway.

Reply:
Hi @jsuarezruiz, this test if failing only on Windows. I don't have a Windows device to work on this. Can you please have a look at this or maybe we can omit Windows for this issue?